trunk/src/constrs/flag_constr_parents.F90

```diff
@@ -0,0 +1,286 @@
+!----------------------------------------------------------------------
+!
+!   routine name       - flag_constr_parents
+!
+!----------------------------------------------------------------------
+!
+!   latest revision    - May 10
+!
+!   purpose            - routine flags ancestors of parents of
+!                        constrained nodes of an element that
+!                        must be unconstrained in constrained
+!                        approximation
+!
+!   arguments :
+!     in:
+!            Mdle      - an element number, same as the middle node
+!                        number
+!
+!----------------------------------------------------------------------
+!
+   subroutine flag_constr_parents(Mdle,Nodesl)
```

Comment:
Did you make any changes to `flag_constr_parents` or only convert to F90?

Reply:
There are a few changes; one of them is that the node marking now marks using bitvisit (so that it can be done over the subdomain and then reduced)

Reply:
Ok, #75 also makes some changes to this routine, adding implicit none, which I'll put directly in here